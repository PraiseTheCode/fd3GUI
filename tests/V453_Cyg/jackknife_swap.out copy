0 1 2 3 4
-0.4092035 0.0389010709 58.6195572 177.071262 184.092975
3.03501755 0.0222219043 18.440300999999977 185.167114 190.235853
1.79745317 0.0538796493 87.46518300000002 180.66454 186.221576
1.20381265 0.0525420003 32.94012900000001 174.221127 215.900333
0.774658417 0.00839467382 350.32384 183.808972 185.489204
0.788545253 0.00788846985 171.428542 183.042526 184.417888
1.0287959 0.0144173712 193.525544 183.523701 184.581477
0.482035004 0.0710143113 326.956096 181.388892 181.695269
