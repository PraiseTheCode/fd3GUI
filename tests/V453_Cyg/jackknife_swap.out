0 1 2 3 4
0.735321924 0.0132538321 349.640821 172.664401 220.322347
1.17468787 0.0139955483 30.721375000000023 175.65569 219.275044
1.37235733 0.0472656279 49.73586499999999 173.141395 216.687788
1.20989523 0.054539149 33.51410899999996 174.394271 215.758842
1.08147445 0.016865307 22.110551999999984 174.263902 220.55232
1.22397482 0.0259782576 35.271713999999974 174.07569 217.797175
1.11329611 0.0152790511 24.7952103 173.712843 211.486801
1.02089209 0.0305979119 17.1937058 174.052155 215.473949
